backstop/tests/table.html

```diff
@@ -165,147 +165,149 @@ <h2>Default</h2>
     <br />
     <section id="demo-default">
       <div class="iui-table">
-        <div class="iui-table-header">
-          <div class="iui-row">
-            <div class="iui-cell iui-slot">
-              <input
-                class="iui-checkbox"
-                type="checkbox"
-                id="select-all-checkboxes"
-              />
-            </div>
-            <div
-              class="iui-cell iui-actionable iui-sorted"
-              tabindex="0"
-              draggable="true"
-              ondragstart="onDragStart(event)"
-              ondragover="onDragOver(event)"
-              ondragleave="onDragLeave(event)"
-              ondrop="onDragLeave(event)"
-            >
-              Name
-              <button
-                class="iui-button iui-borderless iui-filter-button"
-                aria-label="Apply filter"
-                type="button"
-              >
-                <svg-filter-hollow
-                  class="iui-button-icon"
-                  aria-hidden="true"
-                ></svg-filter-hollow>
-              </button>
-              <div class="iui-cell-end-icon">
-                <svg-sort-up
-                  class="iui-button-icon iui-sort"
-                  aria-hidden="true"
-                ></svg-sort-up>
+        <div class="iui-table-header-wrapper">
```

Comment:
This wrapper is needed because sticky won't work with `overflow: hidden` that was on `iui-table-header`.